lib/tessel/setup-3g.js

```diff
@@ -0,0 +1,119 @@
+var Tessel = require('./tessel');
+var log = require('../log');
+
+Tessel.prototype.setup3g = function(options) {
+
+  var apn = options.apn;
+  var dialnumber = options.dialnumber;
+  var username = options.username;
+  var password = options.password;
+  var config = options.config;
+  var on = options.on;
+  var off = options.off;
+  var status = options.status;
+
+  if (config) {
```

Comment:
Similar to the `wifi` and `ap` commands, I would expect configuration to be the default mode of the `3g` command. For example:

```
t2 3g --apn="" --dialnumber="" --username="" --password=""
```

I would expect that command alone to configure the Tessel to connect to that 3g network and be enabled. 

If none of those configuration options are passed in, i.e. `t2 3g`, I would expect to see the current configuration and status of 3g mode. Just like if you use `t2 wifi`, the cli will return the current details of the Tessel's wifi connection (if it is connected).